baselib.py

```diff
@@ -422,8 +422,8 @@ def set_motion_paths_mov(context):
             reader = csv.reader(of, delimiter=' ', skipinitialspace=True)
             for ii in range(3):
                 next(reader)
-            kk = 0
-            jj = 0
+            kk = scene.frame_start
```

Comment:
If a range is selected for the import time (say, between 2 and 6 seconds), this places the keyframes between 2/(freq*time_step) and 6/(freq*time_step), which is okay, it gives greater feedback to the user. 

But the current time is still wrong: it starts from 2 and goes to 6, correctly, but those values are assigned to keyframes between 0 and (6 - 2)/(freq_time_step).